qemu/tests/cfg/sr_iov.cfg

```diff
@@ -29,35 +29,40 @@
             nics = 'nic1 nic2'
             pci_assignable = vf
             image_snapshot = yes
-        - vf_hotplug:
+        - sriov_hotplug:
             type = sr_iov_hotplug
             nics = nic1
             pci_assignable_nic1 = no
             reference_cmd = lspci
             find_pci_cmd = 'lspci | tail -n1'
```

Comment:
Please use `lspci | grep xxx` instead as the device is not always the last one. 
And I prefer to add `match_string` in cfg,  otherwise we have to update sr_iov_hotplug.py(L292) every time we test different cards. Thanks!

Reply:
Sure, I will do it. Thanks !